Objective-C/Tests/URLEndpointListenerTest.m

```diff
@@ -1153,20 +1170,77 @@ - (void) testAcceptOnlySelfSignedCertificateWithPinnedCertificate {
     [self stopListener: listener];
 }
 
-- (void) checkEqualForCert: (SecCertificateRef)cert1 andCert: (SecCertificateRef)cert2 {
-    if (@available(macOS 10.5, iOS 10.3, *)) {
-        CFStringRef cnRef1, cnRef2;
-        AssertEqual(SecCertificateCopyCommonName(cert1, &cnRef1), errSecSuccess);
-        AssertEqual(SecCertificateCopyCommonName(cert2, &cnRef2), errSecSuccess);
-        
-        NSString* cn1 = (NSString*)CFBridgingRelease(cnRef1);
-        NSString* cn2 = (NSString*)CFBridgingRelease(cnRef2);
-        AssertEqualObjects(cn1, cn2);
-    }
-}
-
-- (void) releaseCF: (CFTypeRef)ref {
-    if (ref != NULL) CFRelease(ref);
+- (void) testListenerWithImportIdentity {
```

Comment:
This is the test that tests create identity with identity. Can you also add another test that uses TLSIdentity's importIdentity?